components/status_dropdown/status_dropdown.tsx

```diff
@@ -266,16 +276,16 @@ export default class StatusDropdown extends React.PureComponent {
         );
     }
 
-    refCallback = (ref) => {
+    refCallback = (ref: MenuWrapper): void => {
         if (ref) {
             this.setState({
-                width: ref.state.width,
-                open: ref.state.open,
```

Comment:
hi @devinbinnie , there is no state open, so i rename it to state openUp

Reply:
See above

Reply:
ok, done